test/Sign.Cli.Test/AzureCredentialOptionsTests.cs

```diff
@@ -37,57 +37,87 @@ public void CredentialTypeOption_Always_IsNotRequired()
         }
 
         [Fact]
-        public void ManagedIdentityOption_Always_HasArityOfZeroOrOne()
+        public void TenantIdOption_Always_HasArityOfZeroOrOne()
```

Comment:
```suggestion
        public void TenantIdOption_Always_HasArityOfExactlyOne()
```

Reply:
Thanks for noticing those copy paste mistakes.